protocol/daemons/server/liquidation.go

```diff
@@ -26,32 +24,29 @@ func (server *Server) WithLiquidatableSubaccountIds(
 	return server
 }
 
-// ExpectLiquidationsDaemon registers the liquidations daemon with the server. This is required
-// in order to ensure that the daemon service is called at least once during every
-// maximumAcceptableUpdateDelay duration. It will cause the protocol to panic if the daemon does not
-// respond within maximumAcceptableUpdateDelay duration.
-func (server *Server) ExpectLiquidationsDaemon(maximumAcceptableUpdateDelay time.Duration) {
-	server.registerDaemon(types.LiquidationsDaemonServiceName, maximumAcceptableUpdateDelay)
-}
-
 // LiquidateSubaccounts stores the list of potentially liquidatable subaccount ids
 // in a go-routine safe slice.
 func (s *Server) LiquidateSubaccounts(
 	ctx context.Context,
 	req *api.LiquidateSubaccountsRequest,
-) (*api.LiquidateSubaccountsResponse, error) {
+) (
+	response *api.LiquidateSubaccountsResponse,
+	err error,
+) {
+	// Capture valid responses in metrics.
+	defer func() {
+		if err == nil {
+			s.reportValidResponse(types.PricefeedDaemonServiceName)
```

Comment:
same here, should this be liquidation daemon?

Reply:
Yes, I'm sorry about that.